src/mjolnir/statistics_json.cc

```diff
@@ -1,283 +0,0 @@
-#include <boost/format.hpp>
```

Comment:
hmm what happened to this file?

Reply:
I manually removed this file and `formlocalpath.cc` after tidy logged that there were no compile commands for them. I checked and could see no use of them anywhere.